source/sass/abstracts/_variables.scss

```diff
@@ -68,11 +68,13 @@ $font-sizes: (
 $breakpoints: (
   "mob-portrait" "(min-width: 320px)",
   "mob-landscape" "(min-width: 480px)",
+  // Smaller tablet breakpoint for when using the FRU form
+    "tablet-fundraiseup" "(min-width: 600px)",
```

Comment:
Since the FRU form is very strictly set to ~375px, stacking the elements at the tablet-portrait or tablet-landscape looked awkward. 

I reached out to design and they asked I keep the page layouts side-by-side for as long as possible, so I added this FRU specific breakpoint to use in the SCSS!

Reply:
```suggestion
  "tablet-fundraiseup" "(min-width: 600px)",
```